src/chaat/handler/handler.clj

```diff
@@ -0,0 +1,52 @@
+(ns chaat.handler.handler
+  (:require [ring.util.response :as res]
+            [chaat.model.user :as model.user]
+            [java-time.api :as jt]
+            [chaat.handler.validation :as handler.validation]
+            [chaat.errors :refer [do-or-error]]
+            [chaat.db.utils :as db.utils]))
+
+(defn home
+  "Respond with a greeting/welcome"
+  [request]
+  (res/response "Welcome to chaat"))
+
+(defn health-check
+  "Respond with application status: checks server and db status"
+  [db request]
+  (let [result (db.utils/health-check db)]
+    (if (nil? (:error result))
+      (res/response (str "all systems go: " (jt/instant)))
+      (res/bad-request (str (:error result))))))
```

Comment:
[bug]
[400 Bad Request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400) is for client error. A failed health check is not the client's fault. Perhaps you meant to use [status 500](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500)?

Reply:
Yes, I've updated this. Thanks for also linking the MDN docs (:

I've also removed the try/catch blocks for db operations, and this will allow ring to catch and handle internal server errors.